src/utils.ts

```diff
@@ -131,6 +134,27 @@ export class Utils {
         return serverTokens;
     }
 
+    public static getDirectServerConfigCommand(): string[] | undefined {
+        let url: string | undefined = process.env.JF_URL;
+        let user: string | undefined = process.env.JF_USER;
+        let password: string | undefined = process.env.JF_PASSWORD;
+        let accessToken: string | undefined = process.env.JF_ACCESS_TOKEN;
+
+        if (url) {
+            // Add server config with direct connection details using one of the following methods:
```

Comment:
Unlike in Go, multiline comments are supported in Typescript. 
However, I think it would be prettier to make this comment a [JSDoc](https://jsdoc.app/about-getting-started.html) above getDirectServerConfigCommand. See for example other functions in this file.